content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -49,6 +49,7 @@
 "libGD", "GD is an open source code library for the dynamic creation of images by programmers."
 "libjpeg-turbo","Supports project specific porting efforts of POWER8 ppc64/ppc64le"
 "libvpx","Adds VSX optimizations for libvpx on POWER architecture"
+"LibreHealth","LibreHealth is collaborative community for free & open source software projects in Health IT"
```

Comment:
"r" comes before "v"

Also in the overall projects.csv

Reply:
Yup, please do that!

Reply:
Fixed